### ISSUE TITLE
Total score is not correct due to the weight applied to a course

### ISSUE THREAD

**Issue by matthiasgoossens** [EbbDrop/SmarterSmartchool#11]
The calculation currently adds every result together in overallTotalNumerator and divides through the max possible score (overallTotalDenominator). This calculation is not correct because each course has a certain weight. This weight is not accounted for in the current calculation. (e.g. mathematics has a weight of 4 while a history only has a weight of 2)

**Comment by GameboyVlol** [comment]
yes add this feature

**Comment by RaoufiTech** [comment]
i can add this feature but i think every school has different weighs for the different courses

**Comment by GameboyVlol** [comment]
> i can add this feature but i think every school has different weighs for the different courses

make it so that you can either enter the weight or it detects it by seeing the amount of hours in the planner.

**Comment by RaoufiTech** [comment]
yeah thats a good idea, i will try to add it in the weekend or when I dont have lots of tests

**Comment by GameboyVlol** [comment]
okay!

**Comment by GameboyVlol** [comment]
> yeah thats a good idea, i will try to add it in the weekend or when I dont have lots of tests

any luck yet?

**Comment by Iliannnn** [comment]
Yes, this is expected, hence the disclaimer:

> Deze totalen kunnen afwijken van uw werkelijke resultaten doordat niet altijd alle gegevens gekend zijn.

It still provides a rough estimate of your total score, assuming your school ties the number of hours for a subject to its weight. Typically, more hours mean more tests, and therefore more total points for that subject.

I’m aware this approach isn’t perfect. Adding accurate weights is difficult, as each school uses its own calculation method, and often, students aren’t even aware of the exact system used.

What complicates things further is that some schools (like mine) include both *Formative* and *Summative* evaluations, weighted differently based on continuous assessments versus exams. Not all subjects follow this structure, which throws off the balance; one subject might end up with 300 points, while another has just 20.

It *is* possible to support this kind of customization, but doing so would require a highly flexible system, which could be too complex or overkill for the average user.

@matthiasgoossens @GameboyVlol
